.stylelintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Can we move as set `stylelint.config` in the `.vscode/settings` file? or is there another option?

Reply:
And do we need to disable the built-in linters as per: https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint#disable-vs-codes-built-in-linters-optional?

Reply:
I got the hot tip that:

```
I’d start by extending stylelint-config-sass-guidelines and stylelint-config-prettier
```

Reply:
> And do we need to disable the built-in linters as per: https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint#disable-vs-codes-built-in-linters-optional?

Good idea! Disabled! 

> I’d start by extending stylelint-config-sass-guidelines and stylelint-config-prettier

On further research `stylelint-config-prettier` is depreciated and `sass-guidelines` is indeed in use inside of our current scss plugin! 

> Can we move as set stylelint.config in the .vscode/settings file? or is there another option?

I managed to move our stylelint config file to be inside of webview but I couldn't figure out a way to move stylelint packages inside of `webview`.

I tried to install stylelint packages inside of `webview` via the `stylelint.stylelintPath` config option, but `stylelint` kept getting installed inside the root for some reason, even after moving packages inside of `webview/package.json`, deleting `node_modules`, and starting out of a fresh install. I even tried to revert `yarn.lock` to a previous commit and fresh install the packages but `stylelint` still appeared inside of `/node_modules` instead of `/webview/node_modules`. Kept the packages installed in the root for now but can revisit this later